apps/studio/components/interfaces/SQLEditor/SQLEditor.tsx

```diff
@@ -607,7 +607,7 @@ const SQLEditor = () => {
                   <AiIconAnimation loading={isAiLoading} />
                 </motion.div>
 
-                <AnimatePresence initial={false} exitBeforeEnter>
+                <AnimatePresence initial={false} mode="wait">
```

Comment:
seems like exitBeforeEnter is deprecated with the new version of framer-motion
i ran into a client error to replace it with mode wait locally